File: VB/EF/ExtendAuditEF/ExtendAuditEF.Module/BusinessObjects/MyTask.vb
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl.EF

Namespace dxTestSolution.Module.BusinessObjects

    <DefaultClassOptions>
    Public Class MyTask
        Inherits BaseObject

        Public Overridable Property Subject As String

        Public Overridable Property AssignedTo As Contact
    End Class
End Namespace

File: CS/EF/ExtendAuditEF/ExtendAuditEF.Module/BusinessObjects/MyTask.cs
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl.EF;

namespace dxTestSolution.Module.BusinessObjects;

[DefaultClassOptions]
public class MyTask : BaseObject {
    public virtual string Subject{get;set;}
    public virtual Contact AssignedTo{ get; set; }
}

 

